### DISCUSSION TITLE
[@types/node] fs.Dirent is missing path property

### DISCUSSION THREAD

**Discussion by grundb** [DefinitelyTyped/DefinitelyTyped#65456] [Issues with a @types package] [answered]
The `fs.Dirent` class has a `path` property as of v20.1; see https://nodejs.org/docs/latest-v20.x/api/fs.html#direntpath. This property is missing in [`fs.d.ts`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/2d449a0a77a99844bf507f79a41410faeb099d0d/types/node/fs.d.ts#L216). My guess is @Semigradsky would be the one best equipped to deal with this issue.

[a comment by an automated account is not shown]

**Comment by Semigradsky** [comment, marked as answer]
I will take a look

**Reply by Semigradsky** [reply]
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/65462
